### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by howabsurd**

### CODE REVIEW

File: src/db/migrations/20240629042334-add-table-groupDetail.cjs
Comment (mahavirgoyal32) [inline src/db/migrations/20240629042334-add-table-groupDetail.cjs:1]: make group_id to id it is better

**Review by mahavirgoyal32 — CHANGES_REQUESTED** [review]

File: src/db/migrations/20240629042334-add-table-groupDetail.cjs
Comment (howabsurd) [reply src/db/migrations/20240629042334-add-table-groupDetail.cjs:1]: Changed the tables

File: src/service/group/add.js
Comment (mahavirgoyal32) [inline src/service/group/add.js:1]: its better to make add, delete edit fucntions in different files and export through index.js

File: src/repository/groupRepository.js [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
 
 export const groupRepository = {
-  createGroup: async data => {
-    const { group_name, desc } = data;
-    const newGroup = await Model.GroupDetail.create({
-      group_name,
-      desc,
-    });
+  createGroup: async (data, options = {}) => {
+    const newGroup = await Model.GroupDetail.create(data.body, options);
```
Comment (mahavirgoyal32) [inline src/repository/groupRepository.js:6]: directly do this 
const newGroup = await Model.GroupDetail.create(data, options);
and no need to handle internal server error here 
handle only this in services catch and throw internal server error from there

**Review by mahavirgoyal32 — CHANGES_REQUESTED** [review]

**Pull request merged**